### PR TITLE
Gamified section points

### PR DESCRIPTION
**Pull request by loui981c**
## Description

As a wastepicker, I want to me getting points after completing a section.
https://github.com/orgs/Educado-App/projects/9/views/1?pane=issue&itemId=42475671

## Changes

Added a new screen CompleteSection which is shown after the last lecture or exercise in the screen (LectureSwiperScreen). 

## Related Issues

@BenjaminVilladsen we added a button on the last index of the lectureSwiper but the swiper should contain the exercises as well. #175 

## Checklist

- [X] Code has been tested locally and passes all relevant tests.
- [X] Documentation has been updated to reflect the changes, if applicable.
- [X] Code follows the established coding style and guidelines of the project.
- [X] All new and existing tests related to the changes have passed.
- [X] Any necessary dependencies or new packages have been properly documented.
- [X] Pull request title and description are clear and descriptive.
- [X] Reviewers have been assigned to the pull request.
- [X] Any potential security implications have been considered and addressed.
- [X] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)
section_points

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

Branch: dev

Commit id: deb145b

## Notes for Reviewers

The extra points are just placeholder for now, since the logic for extra points gets implemented in the next sprint.

### CODE REVIEW

**Comment by amaliedilling** [comment]
@sofiiagran do you know why the tests are failing?

**Review by JackSMunn — APPROVED** [review]
There's a lot of deleted files, is this on purpose? If it is, approved, and lets go

File: eml/App.js [hunk 1]
```diff
@@ -198,9 +188,18 @@ export default function App() {
...
               <Stack.Screen
-                name="Section"
+                name={"CourseStack"}
+                component={CourseStack}
+                options={{ headerShown: false }}
+              />
+              <Stack.Screen
+                name={"Section"}
                 component={SectionScreen}
-                initialParams={{ course_id: '' }}
```
Comment (sofiiagran) [inline eml/App.js:203]: reason why removed this?

**Review by sofiiagran — CHANGES_REQUESTED** [review]
Looks goodie good! I have one comment + check new merge conflicts as dev have been updated. Please fix the conflicts and then give me a heads up so i can test it locally :))

File: eml/App.js (hunk 1 above)
Comment (amaliedilling) [reply eml/App.js:135]: shouldn't have been removed, fixed now ;))

File: eml/components/navBar/NavBar.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default function NavBar() {
   return (
     <Tab.Navigator
       testID="navBar" // Make sure you set the testID on the correct element
-      initialRouteName={'Home'}
+      initialRouteName={'Central'}
```
Comment (sofiiagran) [inline eml/components/navBar/NavBar.jsx:21]: Is there a reason for this change?

File: eml/components/profile/LogOutButton.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default function LogOutButton(props) {
       await AsyncStorage.removeItem(LOGIN_TOKEN);
       await AsyncStorage.removeItem(USER_INFO);
 
-      navigation.navigate('Login');
+      navigation.navigate('LoginStack');
```
Comment (sofiiagran) [inline eml/components/profile/LogOutButton.js:23]: Is there a reason for this change?

File: eml/screens/lectures/LectureSwipeScreen.jsx
Comment (sofiiagran) [inline eml/screens/lectures/LectureSwipeScreen.jsx:1]: Why are you guys deleting/changing the work in another group's screen? Is this a mistake? And have you talked with @Mikkelmmc @BenjaminVilladsen

File: eml/screens/lectures/LectureScreen.jsx
Comment (sofiiagran) [inline eml/screens/lectures/LectureScreen.jsx:1]: Are you suppose to add the standard button in here?

**Review by sofiiagran — CHANGES_REQUESTED** [review]
Please see my updated comments :))

File: eml/screens/lectures/LectureSwipeScreen.jsx
Comment (sofiiagran) [reply eml/screens/lectures/LectureSwipeScreen.jsx:1]: Could also be the linting?

**Review by sofiiagran — APPROVED** [review]

**Pull request merged**